src/Admin/AbstractAdmin.php

```diff
@@ -2487,6 +2497,24 @@ public function toString($object)
         return sprintf('%s:%s', ClassUtils::getClass($object), spl_object_hash($object));
     }
 
+    /**
+     * {@inheritdoc}
+     */
+    public function subjectToString(): string
```

Comment:
```suggestion
    public function getSubjectName(): string
```

Reply:
It sounds confusing to me, since the string representation of a subject can be represented by any property, not just for a name related concept.

Reply:
subject**To**String sounds also confusing to me, because you don't provide any parameter. What about `subjectAsString`? :laughing: